### DISCUSSION TITLE
Weblate 进群审核问题

### DISCUSSION THREAD

**Discussion by TUsama** [CFPAOrg/Minecraft-Mod-Language-Package#904] [General]
吵之前先清理一下我们的讨论的基础：
1. 我们是wl群，这个群的目的就是为了除去除了硬性翻译能力以外的技能要求，比如GitHub操作等等，但是打开包自己提取语言文件的能力是否是必要的，这一点还需要商讨。
2. 考核的目的紧跟本群的主要矛盾，也就是翻译能力，所以如无必要，那么无需再次增添过多的题目，就算增添题目，也应是考察翻译能力为主，这里还可以进一步展开。

要解决的问题有以下几个：
1. 进群审核的题目是否需要添加？（题目问题）
2. 在形式上是否需要更为丰富的审核机制？（审核形式问题）
3. 以上两者如何进行添加？并给出具体理由。（具体方案问题）

**Comment by TUsama** [comment]
# 个人意见

## 题目问题

进群审核需要添加题目，因为仅凭当前的题目还无法完全确认加群的人是否具有合格的关于 **Minecraft 翻译**的能力，我个人认为目前还没有照顾/没有完全照顾的点有如下几点：

   - 专有名词问题

      说明：Minecraft  本身有较多的专有名词，需要译者熟悉并且掌握，具体可以参看 Minecraft wiki 并据此出题。

   - 长句子翻译问题

      说明：这个是纯翻译能力的考察，现有试题中没有长状语/定语考察，然而实际上如此的句子在翻译中时常可以看到，而且因为两种语言的差异，对于长状语/定语句子的翻译本身就是重点，所以希望加设一些题目，对这一能力进行考察。

      这里遗留的问题是，是否需要 Chroma 所提到的中译英的题目？这里需要 Chroma 本人提供论证。

## 具体审核题目问题

添加的方向应该围绕我上述给出的两点，具体题目仍需商榷。

##  审核形式问题

考虑到本群的业务颇多：

> 试用期+正常翻译+943+养老院+...都有且只有一个群

   为避免发生剧烈的冲突，同时也为了审核机制的复杂度降低，有必要分群/转移至更为方便的 discord 。

**Reply by 1478599553** [reply]
discord国内使用不方便，首先排除

**Comment by exzhawk** [comment]
审核是为了提高翻译质量, 审核考察翻译能力. 内容能体现翻译能力就好. 翻译能力包括本身的知识以及不会的内容能搜到正确的意思. 最终翻译正确即可.

除翻译能力以外的门槛能拆则拆, 比如怎么找需要汉化的mod, 怎么从mod的jar里扒语言文件等等. 这本来就是weblate的目标之一, 不应列入审核内容.

为防止进群直接养老, 考虑试用期机制, 加群后一段时间(比如三个月), 需要完成若干翻译(比如100条)并得到review权限用户认可.

**Comment by fansi2020** [comment]
我认为不应该追求上限 应该着眼于下限
规定限时翻译多少是不合适的 除非你想商业化。

**Reply by TUsama** [reply]
我不太认为“限时翻译”与商业化之间存在着明显的逻辑联系，我个人认为，只要时间拉得足够长，翻译的量也控制在较少的单位，那么限时翻译就是一种较好的审核方式。

补足一个形式化的逻辑关系论证 ：
1. 在商业化翻译中，因为需要每一个人都进行翻译生产，那么限时限量是合理的，因为在生产过程中有**效率**的要求。
2. 限时翻译，从限时上看，是为了提高**效率**，从“翻译”上看，具有考察玩家**翻译能力**的作用。
3. 如果限时足够长，那么就谈不上“效率”一词。
4. 如果没有“效率”的追求，那么就谈不上“商业化”。
5. 因而基本上只有对于翻译能力的考察。

**Comment by 1478599553** [comment]
我认为：
不需要进行操作（解压jar、了解文件结构的考核）
因为：
1.首先，在我看来，这是比较基本的操作，如果不能掌握，一般不会参与翻译
2.需要给新人成长空间，并且能否进行操作（上面所说）与翻译能力无关，我们不需要这方面的能力或者没有特别需要

但是，我认为目前问卷比较简单，应该加难度。但是为了兼顾943不要太长的想法，只能在内容上面下功夫。或者增加题目类型（中译英、查找考证能力考核等）

综上，考核主要往翻译能力方面靠，但是翻译方面门槛问卷可以适当抬高

**Comment by WuzgXY-GitHub** [comment]
现已有成型工单问卷，但还可以参考这里的部分建议。
